src/services/AssetPoolService.ts

```diff
@@ -68,22 +68,23 @@ export default class AssetPoolService {
                 contract.methods.grantRole(MINTER_ROLE, assetPool.address),
                 assetPool.network,
             );
-        }
 
-        // TODO Move this to a user action from dashboard
-        const adminBalance = await contract.methods.balanceOf(admin.address).call();
-        if (Number(String(adminBalance)) > 0) {
-            await TransactionService.send(
-                contract.options.address,
-                contract.methods.approve(assetPool.address, adminBalance),
-                assetPool.network,
-            );
-            await TransactionService.send(
-                assetPool.contract.options.address,
-                assetPool.contract.methods.deposit(adminBalance),
-                assetPool.network,
-            );
+            const adminBalance = await contract.methods.balanceOf(admin.address).call();
```

Comment:
@valeriagrazzini I guess this could be removed now right? Since the user should be able to deposit an x amount manually. 

Removing it will speed up pool creation quite a bit:)